test/js/valkey/valkey.failing-subscriber.ts

```diff
@@ -13,14 +13,14 @@ function trySend(msg: any) {
   process.send(msg);
 }
 
-let redisUrlResolver: (url: string) => void;
-const redisUrl = new Promise<string>((resolve) => {
+let redisUrlResolver: (msg: { tlsPaths?: { cert: string; key: string; ca: string }; url: string }) => void;
```

Comment:
```suggestion
export interface RedisTestStartMessage {
    ...
};

let redisUrlResolver: (msg: { tlsPaths?: { cert: string; key: string; ca: string }; url: string }) => void;
```

Which you can import in `valkey.test.ts`.